packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -104,16 +93,16 @@ contract VotingV2 is
     mapping(uint256 => Round) public rounds;
 
     // Maps price request IDs to the PriceRequest struct.
-    mapping(bytes32 => PriceRequest) public priceRequests;
+    mapping(bytes32 => PriceRequest) private priceRequests;
```

Comment:
Would really like to keep this public if at all possible. I know this wasn't your first choice, but having visibility into these data structures can just be really useful in a pinch.

Do you see anywhere we could make up this bytecode elsewhere?

Reply:
ok will try